internal/core/metadata/rest_addressable.go

```diff
@@ -331,24 +322,12 @@ func restGetAddressableByAddress(w http.ResponseWriter, r *http.Request) {
 // Notify the associated device services for changes in the device addressable
 func notifyAddressableAssociates(a models.Addressable, action string) error {
```

Comment:
@tonyespy @cloudxxx8 for visibility and in case you have any input.

@jduranf I think I prefer to leave this for now because if, as you say, it doesn't matter to the Device Service when an addressable is modified, then the logic farther down the stack wherein an Addressable's callback URL is called will never fire. This is because, presumably, no callback URL will have been specified. Unless of course, Tony or Cloud +1 your comment for confirmation. Then I'll remove it.

Reply:
I can confirm the Device addressable callback handler is going to be removed.  Thus, we could remove it safely.  The only addressable in Device Service is Device Service itself, and we do not need to notify Device Service when its addressable is changed.

Reply:
Per @cloudxxx8 's confirmation, this has been removed.